asmcomp/branch_relaxation_intf.ml

```diff
@@ -62,6 +62,7 @@ module type S = sig
      the size of out-of-line code (cf. branch_relaxation.mli). *)
   val relax_allocation
      : num_words:int
+    -> blocks:Mach.alloc_info list
```

Comment:
At a first glance, this change seems useless (`blocks` is never used).

Reply:
Indeed. Reverted.

Reply:
Indeed. Reverted.